.github/workflows/main.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run: npm install
       - name: Run Tests
         run: npm run-script test-ci
+        env:
+          NODE_OPTIONS: "--max_old_space_size=4096"
```

Comment:
Does this really require this much memory? If yes, this PR is unmergeable as most people host Nightscout in environments with less than 512 MB of RAM

Reply:
https://github.com/nightscout/cgm-remote-monitor/pull/7820#issuecomment-1375368711